includes/class-amp-autoloader.php

```diff
@@ -101,6 +101,7 @@ class AMP_Autoloader {
 		'AMP_Validation_Callback_Wrapper'    => 'includes/validation/class-amp-validation-callback-wrapper',
 		'AMP_Validated_URL_Post_Type'        => 'includes/validation/class-amp-validated-url-post-type',
 		'AMP_Validation_Error_Taxonomy'      => 'includes/validation/class-amp-validation-error-taxonomy',
+		'AMP_Debug'                          => 'includes/validation/class-amp-debug',
```

Comment:
It's probably debatable whether `class-amp-debug.php` should be in the `validation/` directory. Most of the debugging flags have to do with validation, but some don't, like `disable_amp`.